### PR TITLE
ballot creation wizard now is responsive

### LINKED ISSUE #208: Make creaste-ballot wizard dialog to be responsive
this is just a draft to remind me of this prolemproblem. 
will be updated

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #208

## Description
The current implementation of the wizard for ballot creation was not complete. This PR implements the responsive behavior to fit every possible screen resolution.

## Screen Shots
image
image
image
image

### CODE REVIEW

File: src/pages/trails/ballots/components/BallotForm.vue [hunk 1]
```diff
@@ -708,100 +771,103 @@ q-dialog(
...
+                  br
+                  span {{form.title}}
+                .q-mt-sm
+                  b description:
+                  br
+                  span {{form.description}}
+                .q-mt-sm
+                  b PDF:&nbsp;
+                  span(v-if="form.file") {{filename}}
+                  span(v-if="!form.file") No pdf was attached
```
Comment (donnyquixotic) [inline src/pages/trails/ballots/components/BallotForm.vue:853]: you can just use `v-else` here

File: src/pages/trails/ballots/components/BallotForm.vue [hunk 2]
```diff
@@ -708,100 +771,103 @@ q-dialog(
...
+                .q-mt-sm
+                  .flex.row.gap
+                    .col.flex.column
+                      b options:
+                      ul.q-mt-xs
+                        li(v-for="(option,index) in form.optionsLabels") {{option}}
+                    .col.flex.column
+                      b method:
+                      span(v-if="onlyOneOption") Voters only can choose one option
+                      span(v-if="!onlyOneOption") Voters can choose from {{form.minOptions}} to {{form.maxOptions}} options
```
Comment (donnyquixotic) [inline src/pages/trails/ballots/components/BallotForm.vue:866]: you can just use `v-else` here

File: src/pages/trails/ballots/components/BallotForm.vue [hunk 3]
```diff
@@ -708,100 +771,103 @@ q-dialog(
...
+                .q-mt-sm
+                  b open:&nbsp;
+                  span(v-if="openForVoting") this ballot will be open from now until {{moment(form.endTime).format("MMMM Do YYYY")}}
+                  span(v-if="!openForVoting") this ballot will not open immediately and must be oppened manually in the future
```
Comment (donnyquixotic) [inline src/pages/trails/ballots/components/BallotForm.vue:870]: you can just use `v-else` here

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

File: src/pages/trails/ballots/components/BallotForm.vue (hunk 1 above)
Comment (Viterbo) [reply src/pages/trails/ballots/components/BallotForm.vue]: Uh, my bad! fixed here [d59e224](https://github.com/telosnetwork/app-telos-native/pull/223/commits/d59e22409bbdd61d722885cee0ec899727e2da68)

File: src/pages/trails/ballots/components/BallotForm.vue (hunk 3 above)
Comment (Viterbo) [reply src/pages/trails/ballots/components/BallotForm.vue]: fixed here [d59e224](https://github.com/telosnetwork/app-telos-native/pull/223/commits/d59e22409bbdd61d722885cee0ec899727e2da68)

File: src/pages/trails/ballots/components/BallotForm.vue (hunk 2 above)
Comment (Viterbo) [reply src/pages/trails/ballots/components/BallotForm.vue]: fixed here [d59e224](https://github.com/telosnetwork/app-telos-native/pull/223/commits/d59e22409bbdd61d722885cee0ec899727e2da68)

**Review by donnyquixotic — APPROVED** [review]

**Comment by Viterbo** [comment]
fixed in [913fb3c](https://github.com/telosnetwork/app-telos-native/pull/223/commits/913fb3cadb401de3084782860f6d294e27c33c4e).

image

**Review by karynemayer — APPROVED** [review]

**Pull request merged**